checkov/ansible/checks/graph_checks/PanosPolicyLogSessionStart.yaml

```
@@ -0,0 +1,18 @@
+metadata:
+  id: "CKV_PAN_16"
+  name: "Ensure logging at session start is disabled within security policies except for troubleshooting and long lived GRE tunnels"
+  category: "NETWORKING"
```

```suggestion
  category: "LOGGING"
```
I think logging fits here better 🙂